packages/css/src/components/icon-button/icon-button.scss

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-icon-button {
+  background-color: var(--amsterdam-icon-button-background-color);
+  border: 0;
+  color: var(--amsterdam-icon-button-color);
+  cursor: pointer;
+  outline-offset: var(--amsterdam-icon-button-outline-offset);
+  padding-block: 0.5rem;
+  padding-inline: 0.5rem;
+  touch-action: manipulation;
+
+  &:hover {
+    background-color: var(--amsterdam-icon-button-hover-background-color);
+    color: var(--amsterdam-icon-button-hover-color);
+  }
+}
+
+.amsterdam-icon-button--on-background-light {
```

Comment:
Unsure where this would be used. I’d be inclined to leave it out for now.

Reply:
Maybe we should discuss this with design? It was added quite recently.

Reply:
I’ve asked some questions in Figma. Let’s leave it in.